vtex-apps/cms-demo-blocks
# Initial setup

#### What does this PR do? \*

Add basic setup to the app and add some components to be an example 

** NOTE ** translations are not in the app yet, but I think it's ok for this PR

#### How to test it? \*

This is the code of the `store-theme` to mount this page:

```jsonc
"store.not-found#search": {
    "blocks": [
      "positioned-image#not-found-image",
      "rich-text#not-found",
      "rich-text#not-found-text",
      "positioned-search-bar"
    ]
  },
  "positioned-image#not-found-image": {
    "props": {
      "padding": 5,
      "imageProps": {
        "src": "https://storecomponents.vtexassets.com/assets/vtex.file-manager-graphql/images/vtex_video.f18461fbbcdb082efa603f57df1e9a3b932f11fd___5a826e2949a9f7222997cc982ac39da2.png",
        "width": 300,
        "height": 180
      }
    }
  },
  "rich-text#not-found": {
    "props": {
      "textAlignment": "CENTER",
      "textPosition": "CENTER",
      "text": "**Page not found**",
      "font": "t-heading-3",
      "blockClass": ["pageNotFound", "notFoundText"]
    }
  },
  "rich-text#not-found-text": {
    "props": {
      "textAlignment": "CENTER",
      "textPosition": "CENTER",
      "text": "You may have misstyped the address or the page may no longer be available.",
      "blockClass": "notFoundText"
    }
  }
```

1. [Access the workspace](https://klynger--storecomponents.myvtex.com/shafiuasdhf)
2. Check if everything is ok

You can check the blocks [in the cms too](https://klynger--storecomponents.myvtex.com/admin/app/new-cms/landingPage/edit/new)
![image](https://user-images.githubusercontent.com/8517023/87977036-6d92b580-caa4-11ea-8867-415478af9d65.png)


#### Related to / Depends on \*
